tests/cli/test_cli.py

```diff
@@ -1078,6 +1078,65 @@ def test_upload_default_bundle_store(ctx):
     check_contains(bundle_store_name, _run_command([cl, "info", uuid]))
 
 
+@TestModule.register('store_add')
+def test_store_add(ctx):
+    """
+    Tests of command `cl store add` on different blob storage. (Not test --storage-format yet)
+    1. Specify storage type and url
+    2. Specify url but not specify storage type
+    3. Specify wrong storage type and url: expected exit -1
+    """
+    # Create a new azure_blob bundle store, then delete it
+    bundle_store_name = os.getenv('CODALAB_DEFAULT_BUNDLE_STORE_NAME')
+    blob_id = _run_command(
+        [
+            cl,
+            "store",
+            "add",
+            "--name",
+            bundle_store_name,
+            '--storage-type',
+            'azure_blob',
+            '--url',
+            'azfs://devstoreaccount1/bundles',
+        ]
+    )
+    check_contains("azure_blob", _run_command([cl, "store", "ls"]))
+    _run_command([cl, "store", "rm", blob_id])
+    # TODO: set CODALAB_GOOGLE_APPLICATION_CREDENTIALS and test gcp blob storage
```

Comment:
Since we're just running `cl store add`, we don't actually need to connect to GCS so we don't need to set CODALAB_GOOGLE_APPLICATION_CREDENTIALS here.

Reply:
Ohh yes, so I will add some similar test cases of GCS